example/feature/README.md

```diff
@@ -68,8 +68,8 @@ This is useful for jobs that cannot perform any useful work, such as making prog
    <img src="file/itc-gang4.png" width="900"/>
 
 #### TensorFlow Distributed Training
-1. Use [hived-config-1](file/hived-config-1.yaml).
-2. Submit job [itc-dtf](file/itc-dtf.yaml) to VC2, it will success.
+1. Use [hived-config-2](file/hived-config-2.yaml).
```

Comment:
Why change to hived-config-2

Reply:
each vc in hived-config-1 has only one node, the job needs 1 node plus 1 gpu and cannot be scheduled using hived-config-1